services/lsst-login.rst

```diff
@@ -76,16 +76,19 @@ The Kerberos domain for the ``lsst-login`` servers is ``NCSA.EDU``, so something
 
 .. prompt:: bash $ auto
 
-  kinit username@NCSA.EDU
+  kinit -l 25h -r 7d username@NCSA.EDU
   
   # you may get an error like this: 'kinit: Cannot find KDC for realm "NCSA.EDU" while getting initial credentials';
   # if that's the case, the Kerberos config on the local machine may need to be updated with 'dns_lookup_kdc = true'
 
+  # On a Mac, your Kerberos password may be saved in your keychain.
+  # Use /usr/bin/kinit (instead of any conda kinit you may have in your path) and specify "--keychain" to save.
+  
 .. tip::
 
    **Kerberos Tickets Expire**
 
-   - Your Kerberos ticket on your local machine will expire (generally 25 hours after inititally granted) and need to be renewed, which you can do with ``kinit -R``.
+   - Your Kerberos ticket on your local machine will expire (by default 10 hours but up to 25 hours after initially granted with the ``-l`` option shown above) and need to be renewed, which you can do with ``kinit -R``.
```

Comment:
The default depends on your local kerberos config file. So, there isn't really a default.
```suggestion
   - Your Kerberos ticket on your local machine will expire (e.g. 25 hours after initially granted with the ``-l`` option shown above) and need to be renewed, which you can do with ``kinit -R``.
```

Reply:
I guess the default _is_ actually 10 hours, but this can be configured for up to 25 hours in the `kerb5.conf` file. So, the original is fine.